docs/release/release-1.5.1.rst

```diff
@@ -0,0 +1,9 @@
+=============
+Release 1.5.1
+=============
+
+This release works for Presto versions 0.116 through at least 0.152.1
+
+Bug Fixes
+---------
+* Pin the Docker images used for the product tests to a specific version
```

Comment:
That's my thinking. It doesn't make sense to have a release with no release notes for it.